internal/compiler/builtins.slint

```diff
@@ -448,6 +448,7 @@ export NativeButton := _ {
     property <string> text;
     property <image> icon;
     property <bool> pressed: native_output;
+    property <bool> has-focus: native_output;
```

Comment:
`pressed` and `has-focus` are inconsistently named. How about renaming this property to `focused`?

Reply:
True, but nothing I can change in this commit: That property was there before.